### PR TITLE
fix: add missing section in travel guarantee taxi refund form

### PR DESCRIPTION
**Pull request by jonasbrunvoll**
Closes https://github.com/AtB-AS/kundevendt/issues/19327

This PR adds the missing section in the travel guarantee taxi refund form. A component for enable use of mobile camera to directly upload images is not included. This should be a separate PR.

Skjermbilde 2024-10-28 kl  10 04 18

### CODE REVIEW

File: src/page-modules/contact/travel-guarantee/forms/refundTaxiForm.tsx [hunk 1]
```diff
@@ -27,6 +28,43 @@ export const RefundTaxiForm = ({ state, send }: RefundTaxiFormProps) => {
...
   return (
     <div>
+      <SectionCard
+        title={t(PageText.Contact.travelGuarantee.refundTaxi.taxiReciet.title)}
+      >
+        <Typo.p textType="body__primary">
+          {t(PageText.Contact.travelGuarantee.refundTaxi.taxiReciet.info)}
+        </Typo.p>
+
+        {/*Should here add a component for using camara in phone directly.*/}
```
Comment (adriansberg) [inline src/page-modules/contact/travel-guarantee/forms/refundTaxiForm.tsx:38]: Is this a TODO that is planned?

File: src/translations/pages/contact.ts [hunk 2]
```diff
@@ -257,12 +257,17 @@ export const Contact = {
         'If you are unable to reach your destination by an alternative service, we will refund taxi expenses according to the applicable regulations.',
         'Viss du ikkje kjem fram med eit anna rutetilbod, refunderer vi utlegg til drosje etter gjeldande reglar.',
       ),
-      information: {
+      taxiReciet: {
```
Comment (adriansberg) [inline src/translations/pages/contact.ts:260]: `taxiReciet` → `taxiReceipt`

File: src/page-modules/contact/travel-guarantee/travelGuaranteeFormMachine.ts [hunk 3]
```diff
@@ -263,6 +268,7 @@ export const fetchMachine = setup({
             kilometersDriven: kilometersDriven,
             fromAddress: fromAddress,
             toAddress: toAddress,
+            amaount: amount,
```
Comment (adriansberg) [inline src/page-modules/contact/travel-guarantee/travelGuaranteeFormMachine.ts:271]: `amaount` → `amount`

File: src/page-modules/contact/travel-guarantee/forms/refundTaxiForm.tsx (hunk 1 above)
Comment (jonasbrunvoll) [reply src/page-modules/contact/travel-guarantee/forms/refundTaxiForm.tsx:38]: We haven´t really discussed this feature, but it is included in  `Spec kontaktskjemaer FRAM`, so I choose to say yes.

Comment (adriansberg) [reply src/page-modules/contact/travel-guarantee/forms/refundTaxiForm.tsx:38]:
```suggestion
        {/* TODO: Add a component for using camera in phone directly. */}
```

File: src/page-modules/contact/travel-guarantee/forms/refundTaxiForm.tsx [hunk 4]
```diff
@@ -27,6 +28,44 @@ export const RefundTaxiForm = ({ state, send }: RefundTaxiFormProps) => {
...
   return (
     <div>
+      <SectionCard
+        title={t(PageText.Contact.travelGuarantee.refundTaxi.taxiReceipt.title)}
+      >
+        <Typo.p textType="body__primary">
+          {t(PageText.Contact.travelGuarantee.refundTaxi.taxiReceipt.info)}
+        </Typo.p>
+
+        {/* TODO: Add a component for using camera in phone directly. */}
```
Comment (mortennordseth) [inline src/page-modules/contact/travel-guarantee/forms/refundTaxiForm.tsx:38]: This is already in place.

**Review by mortennordseth — APPROVED** [review]

**Pull request merged**